src/React.Sample.Owin/Content/Index.html

```diff
@@ -0,0 +1,21 @@
+﻿<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <title>React OWIN Sample</title>
+    <script src="react-0.12.2.js"></script>
+    <script src="jquery-1.10.0.min.js"></script>
```

Comment:
Please don't add jQuery to any examples :)

Can you copy the example from `React.Sample.Mvc4`?